cmd/retry.go

```diff
@@ -91,6 +91,8 @@ func newRetryTimer(unit time.Duration, cap time.Duration, jitter float64, doneCh
 	go func() {
 		defer close(attemptCh)
 		nextBackoff := 0
+		// Channel used to signal after the expiry of backoff wait seconds.
+		var backoffWait <-chan time.Time
```

Comment:
What if `doneCh` gets a message when the goroutine blocked on `<-timer.C`? Its still not responsive right ?